### PR TITLE
relay: Make SSL certificate/key loading error handling more verbose

### PR DESCRIPTION
**Pull request by NP-Hardass**

### CODE REVIEW

**Comment by flashcode** [comment]
Hi,
Could you please update your commit and change only source code files, not translations (they're updated later and could cause conflicts like now)?
Thanks.

**Comment by NP-Hardass** [comment]
@flashcode is there something else you need from me regarding this patch?

File: src/plugins/relay/relay-network.c [hunk 1]
```diff
@@ -85,13 +86,22 @@ relay_network_set_ssl_cert_key (int verbose)
...
-                                    _("%s%s: warning: no SSL certificate/key "
-                                      "found (option relay.network.ssl_cert_key)"),
-                                    weechat_prefix ("error"), RELAY_PLUGIN_NAME);
+                                    _("%s%s: GnuTLS error: %s: %s"),
+                                    weechat_prefix ("error"), RELAY_PLUGIN_NAME,
+                                    gnutls_strerror_name(ret),
+                                    gnutls_strerror(ret));
                 }
             }
             free (certkey_path2);
```
Comment (flashcode) [inline src/plugins/relay/relay-network.c:95]: In the else clause, this variable is not freed.
You should move this block after the else block and replace it with:

```C
    if (certkey_path2)
        free (certkey_path2);
```

**Review by flashcode — CHANGES_REQUESTED** [review]

**Comment by flashcode** [comment]
Thanks, merged, with some changes, and adapted the code to the latest version (options "ssl" have been renamed to "tls" in 4.0.0).

**Pull request closed without merging**